test/cpp/end2end/async_end2end_test.cc

```diff
@@ -111,30 +112,35 @@ class Verifier {
     return *this;
   }
 
+  int Next(CompletionQueue* cq, bool ignore_ok) {
```

Comment:
It will help to write a brief comment on when to use Next() vs when to use Verify() functions..